### PR TITLE
SEO-3554 - WPF Controls

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
Please review the changes for blazor controls.

Regards,
Sharly A.

### CODE REVIEW

File: wpf/Maps/Getting-Started.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You can add SfMap reference using one of the following methods:
 
 Syncfusion WPF components are available in [`nuget.org`](https://www.nuget.org/). To add SfMap to your project, open the NuGet package manager in Visual Studio, search for [Syncfusion.SfMaps.WPF](https://www.nuget.org/packages/Syncfusion.SfMaps.WPF), and then install it.
 
-![Adding SfMap reference from NuGet](Getting-Started_images/Adding SfMaps reference.png)
+![Adding SfMap reference from NuGet](Getting-Started_images/Adding-SfMaps-reference.png)
```
Comment (gowthamanthavasiyappan) [inline wpf/Maps/Getting-Started.md:22]: Update proper image name, you need to change the image name in the source location also.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: wpf/Maps/Getting-Started.md (hunk 1 above)
Comment (Sharly-Atieno) [reply wpf/Maps/Getting-Started.md]: modified

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**